vanilla-framework/vanilla-framework
# Add Sass build/watch functionality to Vanilla Docs

## Done

- **Note:** I've created a feature branch `docs-1.7.0` as this PR's base for all the docs-related work going into releasing 1.7.0. I will have a separate PR for fixing up the styling.
- Added `node-sass` and `vanilla-framework` to the /docs packages
- Updated `package.json` to build the css from scss, and build the docs when `./run build` is run
- Fixed the `watch` script so that it rebuilds when `.scss` files are edited
- Moved the inline styles in `template.html` to `/static/sass/etc`

## QA

- Pull code
- `cd docs`
- Run `./run` and check that static/sass/*.scss get built into static/css/styles.css
- Check that the docs are served correctly, with Vanilla 1.7.0-beta-2.1 styling (will update to 1.7.0 when released).
- Open another terminal and run `./run watch`
- Edit `static/sass/styles.scss` and check that the css is rebuilt and the changes occur
